receiver/prometheusexec/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
Standard naming convention for this folder would be `prometheusexecreceiver`.

Also you should do `go mod init` here otherwise this will be included in the base module by default. Probably it would make more sense to implement the receiver skeleton first, even if it does nothing 😄

Reply:
Yes! fixing both thanks